lib/search/query_components/booster.rb

```diff
@@ -22,32 +24,24 @@ def boost_filters
     end
 
     def boosted_formats
-      {
-        # Mainstream formats
-        "service_manual_guide" => 0.3,
-        "service_manual_topic" => 0.3,
-        "smart-answer" => 1.5,
-        "transaction" => 1.5,
+      government_index_config = format_boost_config["government_index"]
+      government_index_boost = government_index_config["boost"]
+      government_formats = government_index_config["formats"]
 
-        # Inside Gov formats
-        "topical_event" => 1.5,
-        "minister" => 1.7,
-        "organisation" => 2.5,
-        "topic" => 1.5,
-        "document_series" => 1.3,
-        "document_collection" => 1.3,
-        "operational_field" => 1.5,
-        "contact" => 0.3,
+      individual_format_boosts = format_boost_config["format_boosts"]
 
-        # Should appear below mainstream content
-        "aaib_report" => 0.2,
-        "dfid_research_output" => 0.2,
-        "hmrc_manual_section" => 0.2,
-        "service_standard_report" => 0.2,
+      boosted_formats = (government_formats + individual_format_boosts.keys).uniq
```

Comment:
Discussed offline: there are a few odd "formats" like "edition" that are present in both indexes at the moment. They seem to be used for static pages, lists, finders, rather than regularly published content.

There aren't many of them, so I think for now we can decide either way (apply the boost or don't) without it affecting many searches.